cdap-app-fabric-tests/src/test/java/co/cask/cdap/internal/app/services/http/handlers/WorkflowHttpHandlerTest.java

```diff
@@ -373,16 +373,14 @@ public void testWorkflowPauseResume() throws Exception {
     verifyRunningProgramCount(programId, runId, 0);
 
     // Verify that Workflow is still suspended
-    verifyProgramRuns(programId, "suspended");
+    verifyProgramRuns(programId, ProgramRunStatus.SUSPENDED);
 
     // Resume the execution of the Workflow
     resumeWorkflow(programId, runId, 200);
 
     // Workflow should be running
     waitState(programId, ProgramStatus.RUNNING.name());
 
-    verifyProgramRuns(programId, "running");
```

Comment:
Is this removed because waitState calls the `/status` endpoint which now fetches the RUNNING or STARTING run records from the store?